sop-extras/CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 # Changelog entries
 
+<a id='changelog-0.2.3.0'></a>
+## 0.2.3.0 -- 2025-04-02
+
+### Non-Breaking
+
+- Refactor `Index` to use `NS ((:~:) x) xs`. Pattern synonyms `IZ` and `IS` are
+  provided, making this change transparent for users.
```

Comment:
Hmm, I realize now that this actually was a breaking change, as some functions require additional constraints

Reply:
Arg indeed.

Reply:
Done